webapp/channels/src/components/admin_console/admin_console.tsx

```diff
@@ -212,23 +204,33 @@ export default class AdminConsole extends React.PureComponent<Props, State> {
             cloud: this.props.cloud,
             isCurrentUserSystemAdmin: this.props.isCurrentUserSystemAdmin,
         };
+
         return (
             <>
                 <AnnouncementBarController/>
                 <SystemNotice/>
                 <BackstageNavbar team={this.props.team}/>
-                <AdminSidebar onFilterChange={this.onFilterChange}/>
+                <AdminSidebar onSearchChange={this.handleSearchChange}/>
                 <div
                     className='admin-console__wrapper admin-console'
                     id='adminConsoleWrapper'
                 >
-                    <Highlight filter={this.state.filter}>
+                    <SearchKeywordMarking
+                        keyword={this.state.search}
+                        pathname={this.props.location.pathname}
+                    >
                         {this.renderRoutes(extraProps)}
-                    </Highlight>
+                    </SearchKeywordMarking>
                 </div>
-                {discardChangesModal}
+                <DiscardChangesModal
+                    show={showNavigationPrompt}
+                    onConfirm={confirmNavigation}
+                    onCancel={cancelNavigation}
+                />
                 <ModalController/>
             </>
         );
     }
 }
+
+export default AdminConsole;
```

Comment:
Why'd you move this down here? I've noticed you and some others have been moving the `export default` away from the component definition in more places, and I'm curious why.

Reply:
i had connect redux before that why had to bring it down, but didnt needed it anymore